src/components/CreateVmWizard/CreateVmWizard.js

```diff
@@ -162,21 +163,27 @@ class CreateVmWizard extends React.Component {
         preventEnter: false,
         preventExit: false,
 
-        render: () => (
-          <Storage
-            id='create-vm-wizards-storage'
-            vmName={this.state.steps.basic.name}
-            clusterId={this.state.steps.basic.clusterId}
-            dataCenterId={this.state.steps.basic.dataCenterId}
-            optimizedFor={this.state.steps.basic.optimizedFor}
-            disks={this.state.steps.storage.disks}
-
-            onUpdate={({ valid = true, ...updatePayload }) => {
-              this.handleListOnUpdate('storage', 'disks', updatePayload)
-              this.wizardStepsMap.storage.preventExit = !valid
-            }}
-          />
-        ),
+        render: () => {
+          const dataCenterId = this.state.steps.basic.dataCenterId
+          const invalidDisk = this.shouldPreventStorageExit()
+          this.wizardStepsMap.storage.preventExit = !!invalidDisk
+          return (
+            <Storage
+              id='create-vm-wizards-storage'
+              vmName={this.state.steps.basic.name}
+              clusterId={this.state.steps.basic.clusterId}
+              dataCenterId={dataCenterId}
+              optimizedFor={this.state.steps.basic.optimizedFor}
+              disks={this.state.steps.storage.disks}
+
+              onUpdate={({ valid = true, ...updatePayload }) => {
+                this.handleListOnUpdate('storage', 'disks', updatePayload)
+                this.wizardStepsMap.storage.preventExit = !valid
```

Comment:
`valid` seems always `true` here

Reply:
It's the default value.
maybe I'm not understanding what's the problem?

Reply:
Sorry for not being clear enough - this comment is linked with my comment [above](https://github.com/oVirt/ovirt-web-ui/pull/1233#discussion_r449618665)

> It seems we should trigger the validations and set here also the valid flag (otherwise it will be always true).

It seems that we should explicitly set `valid` flag there.